packages/experimental/Badge/SPEC.md

```diff
@@ -0,0 +1,330 @@
+   * @defaultvalue circular
+   */
+  shape?: BadgeShape;
+
+  /** Sets style of Badge to a preset size style
+   * @defaultvalue medium
+   */
+  size?: BadgeSize;
+}
+export interface BadgeProps extends BadgeCoreProps {
+  /**
+   * A Badge can have its content and borders styled for greater emphasis or to be subtle.
+   * It can be filled, outline, ghost, inverted
+   * @defaultvalue filled
+   */
+  appearance?: BadgeAppearance;
+
+  /**
+   * A Badge can be one of preset colors
+   * @defaultvalue brand
+   */
+  color?: BadgeColor;
+
+  /*
+   * Source URL or name of the icon to show on the Badge.
+   */
+  icon?: IconSourcesType;
+
+  /**
+   * Icon can be placed before or after Badge's content.
+   * @default before
+   */
+  iconPosition?: BadgeIconPosition;
+}
+```
+
+### Styling Tokens
+
+Tokens can be used to customize the styling of the control by using the `customize` function on the `Badge`. For more information on using the `customize` API, please see [this page](../../framework/composition/README.md). The `Badge` has the following tokens:
+
+```ts
+export interface BadgeCoreTokens extends LayoutTokens, FontTokens, IBorderTokens, IShadowTokens, IColorTokens {
+  /**
+   * Set the bottom edge of the Badge
+   */
+  bottom?: FlexStyle['bottom'];
+
+  /**
+   * The height of the Badge.
+   */
+  height?: number;
+  /**
+   * The icon color.
+   */
+  iconColor?: ColorValue;
+  /**
+   * The icon size.
+   */
+  iconSize?: number;
+  /**
+   * Set the left edge of the Badge
+   */
+  left?: FlexStyle['left'];
+
+  /**
+   * Set the right edge of the Badge
+   */
+  right?: FlexStyle['right'];
+
+  /**
+   * Set the top edge of the Badge
+   */
+  top?: FlexStyle['top'];
+
+  /**
+   * The width of the Badge.
+   */
+  width?: number;
+
+  /**
+   * Sizes of the Badge
+   */
+  tiny?: BadgeTokens;
+  extraSmall?: BadgeTokens;
+  small?: BadgeTokens;
+  medium?: BadgeTokens;
+  large?: BadgeTokens;
+  extraLarge?: BadgeTokens;
+
+  /**
+   * Shapes of the Badge
+   */
+  rounded?: BadgeTokens;
+  circular?: BadgeTokens;
+  square?: BadgeTokens;
+}
+export interface BadgeTokens extends BadgeCoreTokens {
+  /**
+   * The size of the icon.
+   */
+  iconSize?: number;
+
+  /**
+   * The weight of the lines used when drawing the icon.
+   */
+  iconWeight?: number;
+
+  /**
+   * Additional states that can be applied to a Badge
+   */
+  filled?: BadgeTokens;
+  outline?: BadgeTokens;
+  tint?: BadgeTokens;
+  ghost?: BadgeTokens;
+
+  /**
+   * Colors of the Badge
+   */
+  brand?: BadgeTokens;
+  danger?: BadgeTokens;
+  important?: BadgeTokens;
+  informative?: BadgeTokens;
+  severe?: BadgeTokens;
+  subtle?: BadgeTokens;
+  success?: BadgeTokens;
+  warning?: BadgeTokens;
+}
+```
+
+### Accessibility
+
+Basic Badges do not recieve focus and are not accessible. Badges should be given a meaningful description.
+This description will be applied, via `accessibilityLabel` to the element decorated by the Badge.
```

Comment:
> Badges should be given a meaningful description.
This description will be applied, via `accessibilityLabel` to the element decorated by the Badge.

I think this is somewhat confusing, since it feels like you're both suggesting that Badges should have a label and not. I think it would be better to phrase this as "Information about the badge should be added to the element that hosts the Badge through the element's `acccessibilityLabel`